### PR TITLE
DisplayProperties: Highlight current settings

### PR DESCRIPTION
**Pull request by ctfloyd**
The previously selected item will now be highlighted (without focus) in DisplayProperties. Previously it was impossible to tell which items were selected beforehand (desktop background, resolution).

Issue: #556

### CODE REVIEW

File: Applications/DisplayProperties/DisplayProperties.cpp [hunk 1]
```diff
@@ -126,10 +146,11 @@ void DisplayPropertiesWidget::create_frame()
     resolution_list->set_background_color(Color::White);
     resolution_list->set_model(*ItemListModel<Size>::create(m_resolutions));
     resolution_list->horizontal_scrollbar().set_visible(false);
+    resolution_list->set_selection_index(m_selected_resolution_index);
     resolution_list->on_selection = [this](auto& index) {
         m_selected_resolution = m_resolutions.at(index.row());
     };
-
+
```
Comment (bugaevc) [inline Applications/DisplayProperties/DisplayProperties.cpp]:
```suggestion

```

File: Libraries/LibGUI/GListView.cpp [hunk 2]
```diff
@@ -82,6 +82,16 @@ void GListView::mousedown_event(GMouseEvent& event)
     selection().clear();
 }
 
+void GListView::set_selection_index(int index)
+{
+    if (!model())
+        return;
+
+    auto index_as_model = model()->index(index, m_model_column);
```
Comment (bugaevc) [inline Libraries/LibGUI/GListView.cpp]: `index_in_model`?

Comment (bugaevc) [reply Libraries/LibGUI/GListView.cpp]: Why is this even necessary? Whoever would use this would be better off accessing `selection` directly

Comment (awesomekling) [reply Libraries/LibGUI/GListView.cpp]: Indeed, the selection should be accessed directly instead of adding this to `GListView`.

**Comment by awesomekling** [comment]
Hmm, not sure what happened here!

**Pull request closed without merging**